src/components/search/Suggestions/Suggestions.tsx

```diff
@@ -1,40 +1,11 @@
 import { List as UIList } from '@faststore/ui'
+import type { HTMLAttributes } from 'react'
 import React, { forwardRef } from 'react'
 import Button from 'src/components/ui/Button'
 import Link from 'src/components/ui/Link'
-import type { HTMLAttributes } from 'react'
 
 import SuggestionProductCard from '../SuggestionProductCard'
-
-const MAX_SUGGESTIONS = 10
-const MAX_SUGGESTIONS_WITH_PRODUCTS = 5
-const MAX_SUGGESTED_PRODUCTS = 4
-const SUGGESTED_PRODUCTS = [
-  {
-    name: 'Ergonomic Wooden Bacon',
-    listPrice: 72.06,
-    price: 46.26,
-    image: [
-      {
-        alternateName: 'rerum',
-        url: 'http://storeframework.vtexassets.com/arquivos/ids/167285/ut.jpg?v=637753017045600000',
-      },
-    ],
-  },
-  {
-    name: 'Handcrafted Rubber Sausages',
-    listPrice: 59.57,
-    price: 32.83,
-    image: [
-      {
-        alternateName: 'ea',
-        url: 'http://storeframework.vtexassets.com/arquivos/ids/155949/voluptas.jpg?v=637752878341070000',
-      },
-    ],
-  },
-]
-
-const SUGGESTIONS = ['Sony MX', 'Sony MV-100 Headphone', 'Sony M2000 Earbuds']
+import './suggestions.scss'
```

Comment:
👀

Reply:
![Screen Shot 2022-03-31 at 14 04 01](https://user-images.githubusercontent.com/10627086/161110728-d51105f0-da55-4a23-bbba-2f68beace00f.png)

Maybe this is a bug by rebase? idk

Reply:
no, after #434 we need to import it directly on the stylesheet